src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -263,6 +277,18 @@ get_tiles(const xrt_core::device* device, const std::string& graph_name)
   return ::get_tiles(aie_meta, graph_name);
 }
 
+int
+get_graph_id(const xrt_core::device* device, const std::string& graph_name)
+{
+  auto data = device->get_axlf_section(AIE_METADATA);
+  if (!data.first || !data.second)
+    return -1;
```

Comment:
Is this a valid return id?  I assume it is not an error, but what does -1 indicate?  Should it be a symbolic constant, or at least document in aie_parser.h:get_grap_id().

Reply:
Defined a symbolic constant,
xrt_core::edge::aie::NON_EXIST_ID
Does it look good?